src/index.html

```diff
@@ -22,6 +22,387 @@
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="Computer"
+          class="card__image"
+        />
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="reviews-stars">
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem stars-aitem--not-active"></div>
+          </div>
+          <p class="reviews-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="Computer"
+          class="card__image"
+        />
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="reviews-stars">
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem"></div>
+
+            <div class="stars-aitem stars-aitem--not-active"></div>
+          </div>
+          <p class="reviews-text">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__link"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
```

Comment:
The image source path 'images/imac.jpeg' should be verified to ensure the image is correctly located in the specified directory. If the image does not display, check the path and file name.